cypress/support/commands/commands.ts

```diff
@@ -59,3 +59,19 @@ Cypress.Commands.add('iFrame', () => {
     .should('not.be.empty')
     .then((body) => cy.wrap(body));
 });
+
+/**
+ * Create resource name
+ */
+const runTimestamp = +new Date();
+
+Cypress.Commands.add('createE2EResourceName', (context) => {
+  return cy.wrap(`e2e-test-${ runTimestamp }-${ context }`);
+});
+
+/*
+* Wait for loading indicator to disappear
+*/
+Cypress.Commands.add('waitForLoadingIndicator', () => {
```

Comment:
Yes! Thanks for the call out. I've created a `LoadingPo` and used it directly where needed instead of using the custom command. Updated in https://github.com/rancher/dashboard/pull/10384/commits/6b54ff7c5da1a2a89e8f05bcef07068aba110629